wmfdata_tests/tests.py

```diff
@@ -171,6 +171,20 @@ def test_load_csv():
 
     log_test_passed("Load CSV to Data Lake")
 
+def test_sql_tuple():
+    try:
+        wmf.utils.sql_tuple([])
+    # We want a ValueError in this case
+    except ValueError:
+        pass
+    else:
+        raise AssertionError("Passing an empty iterable to sql_tuple should raise a ValueError.")
+
+    t = wmf.utils.sql_tuple(("enwiki", "arwiki", "dewiki"))
+    assert t == "('enwiki', 'arwiki', 'dewiki')"
```

Comment:
You covered empty case and multiple elements case, which is great. It would have also been nice to cover the single element case (which, as you had alluded elsewhere, would expose your non-ANSI SQL edge case).

Reply:
Well, the main component of the edge cases (since I now noticed another one) is not having a single element but having an element than includes a single quote or a backslash. I could add tests for those but they would always fail, so that doesn't seem very helpful. Is there something I'm missing?

Reply:
> I could add tests for those but they would always fail

Actually, failing tests are useful too. They document limitations of the implementation. But we can get into those once we add a unit test suite.